rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -27,18 +27,18 @@ mod system_tray_menu;
 #[cfg(target_os = "linux")]
 #[path = "gui/os_linux.rs"]
 #[allow(clippy::unnecessary_wraps)]
-mod os;
+pub(crate) mod os;
```

Comment:
What if you do `use pub(crate) os::set_autostart` instead? Or were you trying to avoid too many stubs?